.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- Kindly, add `test.md` to `.gitignore` according to the [instruction](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)